src/spec.md

```diff
@@ -79,6 +79,49 @@ Each `update` which is passed to the callback comes with the following propertie
 - `update.summary`: optional, short text, shown beside icon (see [`sendUpdate()`](#sendupdate))
 
 
+### sendToChat()
+
+```js
+promise = window.webxdc.sendToChat(message);
+```
+
+`sendToChat()` allows an webxdc app to prepare a message
+that can then be sent to a chat by the user.
+Implementations may ask the user for a destination chat
+and then set up the message as a draft,
+so it is clear that the outgoing message is really a result of some user interaction.
+
+- `message`: an object with the following properties:
+  - `message.file.base64`: base64 encoded file data
+  - `message.file.name`: name of the file
+  - `message.text`: message text that should be sent together with the file
+
+Either `message.file`, `message.text` or both needs to be set.
+The text can usually be modified by the user and
+the user may decide to send the text without the file
+or send nothing at all.
+
+To let the user focus on sending the message,
```

Comment:
"This function will immediately close the webxdc app so that the user can send the message to a chat."

Reply:
to get rewording suggestions commited, it is helpful to point out the reason, it is not always obvious. eg. here, for me, the sentence is shorter, but tunes down reasoning. both are probably good enough, idk, probably not woth many discussions.